scripts/prepublish.js

```diff
@@ -18,6 +18,8 @@ function applyPatches() {
         shell.sed('-i', '#include "zookeeper_log.h"', '#include "zookeeper_log.h"\n#include "winport.h"\n', `${destination}/zk_log.c`);
         shell.sed('-i', '#include "zookeeper.h"', '#include "winport.h"\n#include "zookeeper.h"\n', `${destination}/zk_adaptor.h`);
         shell.sed('-i', '#include "zk_adaptor.h"', '#include "zk_adaptor.h"\n#include "winport.h"\n', `${destination}/zookeeper.c`);
+    } else if (env.isLinux) {
+        shell.exec(`patch -d ${env.rootFolder} -p0 --forward < ${env.workFolder}/no-fipsmode.patch`);
```

Comment:
would it make sense to apply the patch for OS X too? In that case, an `else` could be enough here.

Reply:
Hey @DavidVujic , sorry for the late reply. Unfortunately I don't have an opportunity to test on MacOS but somehow I'd expect it would have been caught earlier if it didn't install there as well ;). My thought here was to make the impact of the change as small as possible. Since the change is pretty harmless, you could consider to apply the patch in any case just to be sure and even get rid of the else.

Reply:
Ok, I'll apply the patch and can test Mac OS X myself in a separate PR. I'll try to do this within the next couple of days. Great work with finding the cause in the C code! 

I would suggest that you also make a PR to the official Zookeeper repo with this patch, because it will probably cause problems for any C Client user running a new version of Openssl.